tests/BrowsershotTest.php

```diff
@@ -83,6 +84,10 @@
     Browsershot::html('<h1><img src="file://" /></h1>');
 })->throws(HtmlIsNotAllowedToContainFile::class);
 
+it('will not allow JS import links not in whitelist', function () {
```

Comment:
Could you also add the reverse test: so add a link in the `allowList` and verify that no exception is thrown.